fundingmanager.go

```diff
@@ -1416,14 +1493,14 @@ func (f *fundingManager) sendFundingLockedAndAnnounceChannel(
 	nextRevocation, err := channel.NextRevocationKey()
 	if err != nil {
 		fndgLog.Errorf("unable to create next revocation: %v", err)
-		return
+		return err
```

Comment:
do `return fmt.Errorf("unable to create next revocation: %v", err)` here and other places in this method